Packs/Phishing/Playbooks/Phishing_-_Generic_v3_6_8.yml

```diff
@@ -3534,11 +3530,27 @@ tasks:
         -  VirusTotal
         - Anomali ThreatStream
         - Hatching Triage
+      playbookName: Detonate URL - Generic v1.5
+      type: playbook
+      iscommand: false
+      brand: ""
     nexttasks:
       '#none#':
       - "52"
+    scriptarguments:
+      URL:
```

Comment:
check this out - maybe @idovandijk removed that intentionally...

Reply:
it is not removed, it just changed places because of the formatting